build.gradle

```diff
@@ -133,6 +133,7 @@ ext {
   MOPUB_MEDIATION_UNITYADS_VERSION = '3.0.1.0'
   APPNEXT_SDK_VERSION = '2.4.4.472'
   INMOBI_SDK_VERSION = '7.2.7'
+  AD_MOB_SDK_VERSION = '17.2.0.0'
 
```

Comment:
According to the documentation, we should also have the play services version 17.2.0. Also, there is a new update of ad mob, 17.2.1.0, and if you update to this version, you should also have 17.2.1.0 play services version